### DISCUSSION TITLE
Which package provide glslc?

### DISCUSSION THREAD

**Discussion by ghost** [msys2/MINGW-packages#16524] [Q&A] [answered]
I'm trying to compile https://github.com/gan74/Yave on MSYS2, cmake failed at configuration with `glslc not found in Vulkan SDK`.

**Comment by Biswa96** [comment, marked as answer]
glslc.exe is provided by shaderc package. You can use pacman to find which package provides a file. For example,

```
x@y UCRT64 ~
$ pacman -F $MINGW_PREFIX/bin/glslc.exe
ucrt64/bin/glslc.exe is owned by ucrt64/mingw-w64-ucrt-x86_64-shaderc 2023.3-1
```
